Makefile

```diff
@@ -29,6 +29,22 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+postman:
+	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
+	# and if needed a token.
+	# For example: make postman env=test token=<MY_TOKEN>
+	docker-compose -f docker-compose.test.yml build --no-cache postman_test
+	docker-compose -f docker-compose.test.yml run --rm postman_test test/$(env).postman_environment.json --global-var "token=$(token)"
+
+
+postman-local:
```

Comment:
Why do we need `postman-local` target?  Can't we just use the `postman` target for both situations?
For example, when running in scenario where token is not needed: `make postman env=api`
And when running in scenario where token is needed: `make postman env=test token=<MY_TOKEN>`

Reply:
added in an ifeq statement to make this more responsive and only require a single command